modules/nf-core/seqtk/sample/main.nf

```diff
@@ -0,0 +1,57 @@
+process SEQTK_SAMPLE {
+    tag "${meta.id}"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/seqtk:1.4--he4a0461_1' :
+        'biocontainers/seqtk:1.4--he4a0461_1' }"
+
+    input:
+    tuple val(meta), path(reads), val(sample_size)
+
+    output:
+    tuple val(meta), path("*.fastq.gz"), emit: reads
+    path "versions.yml"                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    if (!(args ==~ /.*-s[0-9]+.*/)) {
+        args += " -s100"
+    }
+    if ( !sample_size ) {
+        error "SEQTK/SAMPLE must have a sample_size value included"
+    }
+    """
+    printf "%s\\n" $reads | while read f;
+    do
+        seqtk \\
+            sample \\
+            ${args} \\
+            \$f \\
+            ${sample_size} \\
+            | gzip --no-name > ${prefix}_\$(basename \$f)
```

Comment:
Isn't it the `--to-stdout`/`-c` flag that is preferable to be used here? :thinking:

Reply:
Well, it seems neither of them would do much in this circumstance, so I don't think it matters.

Reply:
As in 
```
gzip --no-name -c > ${prefix}_\$(basename \$f)
```